nservicebus/azure-storage-queues/configuration.md

```diff
@@ -60,3 +60,10 @@ This feature can be enabled by specifying `.UseAccountNamesInsteadOfConnectionSt
 snippet:AzureStorageQueueUseAccountNamesInsteadOfConnectionStrings
 
 NOTE: This feature is not available in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Versions 6 and below.
+
+## FIPS compliance
```

Comment:
There was a discussion where-there we support FIPS or not. @DavidBoike do you remember what was the decisions? I wonder because FIPS part could be a general `include` to be used by everything that does support the compliance.